Format and content enhance README.md

janekdb opened the pull request:
Replace bullet points with tables for
- Review labels
- Command overviews

Rescue sbt section from midst of Ant section.

Syntax correct use of ANT_OPTS.

Synchronise IntelliJ menu path to 14.1.

Update OSS repo name. The Central Repository was formerly known
as Maven Central.

Standardise on @-mention instead of @mention.

Mark unfinished section with TODO.

Include a few other minor improvements.

janekdb commented:
Review by @heathermiller, @dickwall, @som-snytt.

som-snytt commented on README.md:
```diff
@@ -91,13 +91,16 @@ To help you plan your contributions, we communicate our plans on a regular basis
 ## Reviewing
 
-Once you've gained some experience with the code base and the process, the logical next step is to offers reviews for others's contributions. The main goal of this whole process, in the end, is to ensure the health of the Scala project by improving the quality of the code base, the documentation, as well as this process itself. Thank you for doing your part!
+Once you've gained some experience with the code base and the process, the logical next step is to offers reviews on the contributions of others. The main goal of this whole process, in the end, is to ensure the health of the Scala project by improving the quality of the code base, the documentation, as well as this process itself. Thank you for doing your part!
```
`offer reviews of` the contributions of others. Is that really one long line? That is one really long line.

janekdb replied:
This paragraph is now split in two with the first sentence simplified.

som-snytt replied:
BTW, I meant one EOL, not one sentence. To remind myself to clarify in future.

janekdb committed:
Format and content enhance README.md

Replace bullet points with tables for
 - Review labels
 - Command overviews

Rescue sbt section from midst of Ant section.

Syntax correct use of ANT_OPTS.

Synchronise IntelliJ menu path to 14.1.

Update OSS repo name. The Central Repository was formerly known
as Maven Central.

Standardise on @-mention instead of @mention.

Mark unfinished section with TODO.

Include a few other minor improvements.

janekdb commented:
Build error root cause:

https://scala-ci.typesafe.com/job/scala-2.11.x-integrate-ide/1399/console

```
Downloaded: http://snapshots.repository.codehaus.org/org/sonatype/oss/oss-parent/7/oss-parent-7.pom (78 B at 0.2 KB/sec)
[ERROR] The build could not read 1 project -> [Help 1]
org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
```

Testing the URL in the browser,
http://snapshots.repository.codehaus.org/org/sonatype/oss/oss-parent/7/oss-parent-7.pom

```
This domain does not resolve - so you should not even be able to get this file
```

Futher digging,

http://www.codehaus.org/ -  _All Codehaus services have now been terminated._ 
http://www.codehaus.org/mechanics/maven/ - _Codehaus no longer serves up Maven repositories,_

janekdb commented:
@adriaanm - infrastructure issue.

Infra issue tracked at scala/scala-jenkins-infra#116

SethTisue commented:
/rebuild

adriaanm commented:
LGTM, thanks! Sorry about the delay.
